examples/scaling-criteo/03-Training-with-TF.ipynb

```diff
@@ -72,7 +72,7 @@
     "# External dependencies\n",
```

Comment:
<p>Need to update the title and probably talk about how this functionality has moved.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
If we want to leave this until a future PR and just merge this that's fine as well.

Reply:
Yeah, I'm trying to do a quick fix for issues raised on this repo, but I agree that the examples do need some attention and rework.